core/src/modules/apiData.ts

```diff
@@ -1,13 +1,16 @@
+import { Api } from "actionhero";
+
 export namespace APIData {
   export function ensureObject(
     param: { [key: string]: any } | string,
+    api?: Api,
     recursing = false
   ) {
     if (!param) {
       return null;
     } else if (Array.isArray(param)) {
       try {
-        return param.map((row) => APIData.ensureObject(row, true));
+        return param.map((row) => APIData.ensureObject(row, api, true));
```

Comment:
So is this now including something that was missed?

Reply:
Yeah - the test I added was failing before because there already was a second param being passed into all these formatters - the `api` object.  That should be changed in actionhero...

Reply:
https://github.com/actionhero/actionhero/pull/2012 is the change in actionhero to prevent this kind of thing in the future